### ISSUE TITLE
Update PyMISP submodule v2.4.165.1

### ISSUE THREAD

**Issue by github-germ** [MISP/MISP#8772]
Our docker builds clone MISP and then pull PyMISP via submodule.

The "official" git tag in PyMISP was bumped to [v2.4.165.1](https://github.com/MISP/PyMISP/tree/v2.4.165.1/pymisp) (commit [320957](https://github.com/MISP/PyMISP/commit/320957f10c1fb8e742295127e4c235ff1b6566d9)), but seems like not the MISP submodule?

With the latest build of our containers we appear to be getting PyMISP v2.4.165 which has the wrong `__version__` in `__init__.py`
```
git submodule status | grep PyMISP
 2b20d84b101a6e5baa1b0fb470f715c31b092c86 PyMISP (v2.4.165)
```
Now, all our PyMISP client apps trigger:
```
WARNING The version of PyMISP recommended by the MISP instance (2.4.165) is newer than the one you're using now (2.4.162.1). Please upgrade PyMISP
```

**Comment by Rafiot** [comment]
Right, we didn't bump that on the 2.4 branch, but if you install pymisp, you will get the right version. Are all your script using PyMISP from within the MISP repo? 

Eitherway, this is an issue for the MISP repo itself, moving it over there.
